### PR TITLE
Created Entity Framework Core

### PR DESCRIPTION
**Pull request by orassr**
Briefly introduction to  Entity Framework Core and fundamentals of ORM.

---

## ✅️ By submitting this PR, I have verified the following

- [] Added descriptive name to PR
  - Your PR should NOT be called `Update index.md` since it does not help the maintainer understand what has been changed.
  - For example, if you create a **Variables** article inside the **Python** directory, the pull request title should be **Python: add Variables article**.
  - Other examples are **Git: edit Git Commit article** or **PHP: create PHP section and add Data Structures article**
- [] Reviewed necessary formatting guidelines in [`CONTRIBUTING.md`](https://github.com/freeCodeCamp/guides/blob/master/CONTRIBUTING.md).
- [] No plagiarized, duplicate or repetitive  content that has been directly copied from another source.

### CODE REVIEW

**Comment by joshalling** [comment]
@orassr It looks like it is still failing because of your folder name. It should be all lowercase and formatted like this `entity-framework-core`. Also, if you are ever curious as to why your Travis CI build is failing, click on the details to the right and it will show you a log with more information.

File: src/pages/csharp/EntityFrameworkCore/index.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+---
+Entity Framework Core : Introduction
```
Comment (amandeepmittal) [inline src/pages/csharp/EntityFrameworkCore/index.md]: Please follow correct Markdown formatting. Add `title:` before `Entity Framework Core`.

File: src/pages/csharp/EntityFrameworkCore/index.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+![logo](https://www.ryadel.com/wp-content/uploads/2015/03/entity-framework-logo.jpg)
+
+*** Will be focused on EF Core 2.X and later ***
+
+Entity Framework or EF is an Object/Relational Mapping (O/RM) framework, 
+based on the ADO.NET and simplify tasks like sooring and accessing data in the Database.
+
+EF is a cross-platform and open source framework.
+
+[EF Core On GitHub] (https://github.com/aspnet/EntityFrameworkCore/)
```
Comment (amandeepmittal) [inline src/pages/csharp/EntityFrameworkCore/index.md]: The links should have spaces in-between [...] and (its link).

File: src/pages/csharp/EntityFrameworkCore/index.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+[EF Core Official Documentation] (https://docs.microsoft.com/ef/core/)
+
+Support databases available :
```
Comment (amandeepmittal) [inline src/pages/csharp/EntityFrameworkCore/index.md]: This could be an unordered list with help of `*` or `-`.

**Comment by joshalling** [comment]
@orassr I am closing this as the requested changes were not made. Please don't hesitate to make the changes and reopen this if you want to contribute.

**Pull request closed without merging**